examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Lettre Examples
+ This folder contains examples showing how to user lettre in your own projects.
+
+## Examples
+- **SMTP** - Sending an email over an unencrypted SMTP connection.
```

Comment:
This should probably specify it's using a local relay on port 25

Reply:
Ah, yes that is much better! Was wondering why the example was unencrypted (aside from accessing legacy mail systems.)

Thanks for pointing out that detail!